app/controllers/application_controller.rb

```diff
@@ -50,8 +50,11 @@ def creating?
   end
   helper_method :creating?
 
-  def set_locale
-    I18n.locale = I18n.default_locale
+  def subdomain_locale_override
+    locale = request.subdomain
+    if I18n.available_locales.include?(locale.to_sym)
+      I18n.locale = locale
+    end
```

Comment:
```suggestion
    else
      I18n.locale = I18n.default_locale
    end
```

Reply:
I think you still need to fall back to setting the default locale in the case where the requested locale / subdomain isn't in our list of available locales.